src/app/tests/suites/certification/Test_TC_CC_3_1.yaml

```diff
@@ -243,14 +243,24 @@ tests:
               - name: "ms"
                 value: 10000
 
+    #https://github.com/CHIP-Specifications/chip-test-scripts/issues/493#issuecomment-1422075950
     - label: "TH reads CurrentHue attribute from DUT"
-      PICS: CC.S.F00 && CC.S.A0000
-      command: "readAttribute"
-      attribute: "CurrentHue"
-      response:
-          constraints:
-              minValue: 166
-              maxValue: 224
+      PICS: CC.S.F00 && CC.S.A0000 && PICS_SKIP_SAMPLE_APP
```

Comment:
@yinyihu-silabs I think the problem with these 15% variance cases is , any time the step goes in a decreasing order and goes to 0. The variance of 15% could be( 0-15 or 255-240 since its an int value)  , currently in Yamls we do not have a capability to add 2 range checks for a particular condition. So leaning towards the solution for 1.1 that in case a device/platform requires the 15% variance might have to execute manually. After 1.1 we must sit together and fix all such cases in a much more systematic way.

Reply:
@yinyihu-silabs We don't have support for multiple range checks in YAML for step 3b.
https://github.com/project-chip/connectedhomeip/issues/23197